src/apps/chifra/internal/daemon/routes.go

```diff
@@ -213,7 +214,31 @@ var routes = Routes{
 		HandleWebsockets(connectionPool, w, r)
 	}},
 	Route{"Index", "GET", "/", Index},
-	Route{"EditName", "POST", "/names", EditName},
+	Route{"CreateName", "POST", "/names", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Is the EditName routine needed any more? Can we remove it and all associated calls internal to it? I will write an issue and clean it up after the merge.

Reply:
Sorry, I forgot to do it. We can remove it